vite.config.js

```diff
@@ -0,0 +1,20 @@
+import {defineConfig} from 'vite'
+
+export default defineConfig({
+    build: {
+        outDir: '..',
+        rollupOptions: {
+            output: {
+                assetFileNames: '[name].[ext]',
+            }
+        },
+    },
+    root: 'src',
+    server: {
```

Comment:
@francoisfreitag Any idea to serve files from /images ?

Reply:
Very new to Vite.js, but from [Static Asset Handling](https://vitejs.dev/guide/assets.html#importing-asset-as-url), it expects images to be referenced from a JS file in order to bundle it. I see two solutions:
1. Let Vite know that all files under `images` are public. This can be best achieved by changing the hierarchy, moving images to a new `public/` directory. (my preference, see https://github.com/mmouterde/ultimate-thorigne-fouillard/pull/17)
2. Create an index.js file that sets the `src` attributes on images and have Vite bundle the images, similarly to the documented example from https://vitejs.dev/guide/assets.html#importing-asset-as-url. Extra work, not worth it IMO.

Not sure how the web server is configured, but ideally, it would serve a `dist` directory to prevent access to source and build files, instead of serving from the root of the repository.

Reply:
Note that the issue already exists, I don’t think http://www.ultimate-thorigne-fouillard.fr/gulpfile.js is meant to be served by the web server.